Packages/tests/UTF_HardwareHelperFunctions.ipf

```diff
@@ -117,6 +117,14 @@ Function TEST_CASE_END_OVERRIDE(name)
 	string dev, experimentNWBFile, baseFolder, nwbFile
 	variable numEntries, i, fileID, nwbVersion, expensiveChecks
 
+	// be sure that DAQ/TP is stopped before we do anything else
+#ifndef TESTS_WITH_NI_HARDWARE
```

Comment:
@t-b so if it is **not** TESTS_WITH_NI_HARDWARE, we stop DAQ/TP when the testing timer times out?

Also, the testing timer is in Bamboo, not the .pxp test file, is that correct?

Reply:
This is not the bamboo timer. This is the testing framework timeout for the reentry function. See also https://docs.byte-physics.de/igor-unit-testing-framework/advanced.html#test-cases-with-background-activity.

The hardware test cases are always in two parts:
- setup, starting DAQ/TP
- checking results 

The "checking results" step is in the functions name `XXX_REENTRY`. This part is called once DAQ/TP is finished *or* when the user settable timeout hits. We have set this timeout to 10m. Once that timeout is hit the `XXX_REENTRY` function is called and *then* we call TEST_CASE_END_OVERRIDE. And at this point DAQ/TP could still be ongoing, so we have to stop it before we do anything else.

Reply:
Yes this part is only executed for ITC hardware.